src/gift-fill.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24"><defs><path d="M11 14v8H5c-1.057 0-1.922-.482-1.995-1.813L3 20v-6h8zm10 0v6c0 1.409-.82 1.952-1.857 1.997L19 22h-6v-8h8zM5 2h1c2.89 0 4.87 1.239 6 3.28C13.13 3.238 15.11 2 18 2h1v1c0 .891-.211 2.158-.856 3.447a6.645 6.645 0 01-1.064 1.555L20 8a2 2 0 011.995 1.85L22 10v2h-9V9h-2v3H2v-2a2 2 0 011.85-1.995L4 8l2.92.002a6.645 6.645 0 01-1.064-1.555c-.599-1.197-.824-2.374-.853-3.25L5 3V2h1zm11.86 2.09c-1.568.262-2.584 1.093-3.216 2.357-.185.37-.327.763-.432 1.163l-.072.3c1.568-.262 2.584-1.093 3.216-2.357.23-.463.396-.96.504-1.464zm-9.72 0c.108.503.273 1 .504 1.463.597 1.193 1.537 2.002 2.96 2.309l.256.049-.072-.301a6.005 6.005 0 00-.432-1.163C9.724 5.183 8.708 4.352 7.14 4.09z" id="a"/></defs><use fill="#000" fill-rule="nonzero" xlink:href="#a"/></svg>
```

Comment:
- Please remove unnecessary `<defs>` `</defs>`
- Remove the`id="a"`
- Remove `<use fill="#000" fill-rule="nonzero" xlink:href="#a"/>`